Multiprotocol/Multi_Protos.ino

```diff
@@ -171,7 +171,7 @@ const char STR_SUBTYPE_KYOSHO2[] =    "\x05""KT-17";
 const char STR_SUBTYPE_FUTABA[] =     "\x05""SFHSS";
 const char STR_SUBTYPE_JJRC345[] =    "\x08""JJRC345\0""SkyTmblr";
 const char STR_SUBTYPE_MOULKG[] =     "\x06""Analog""Digit\0";
-const char STR_SUBTYPE_KF606[] =      "\x06""KF606\0""MIG320";
+const char STR_SUBTYPE_KF606[] =      "\x07""KF606\0 ""MIG320\0""ZCZ50  ";
```

Comment:
Why have you changed the length to 7 and padded with spaces ?
It should be: const char STR_SUBTYPE_KF606[] =      "\x06""KF606\0""MIG320""ZCZ50\0";

Reply:
Because "MIG320" was last and did not need "\0" at the end. Now it does, so I needed to extend the length by one. Shorter string are padded too at different places https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/blob/b957f8bae7241f9fa48619e22fe592c510f8b308/Multiprotocol/Multi_Protos.ino#L134, so I assume this is way to go.

Reply:
Why do you think that mig320 needs a \0 ? It does not need it at the beginning, middle or end since the string is max 6 bytes long... Please try the way described and you'll see that it will work just fine. As soon as this is corrected, I'll merge the PR.

Reply:
You are correct. It's fixed now.